src/source/PeerConnection/SessionDescription.c

```diff
@@ -295,8 +295,10 @@ STATUS setTransceiverPayloadTypes(PHashTable codecTable, PHashTable rtxTable, PD
             }
         }
 
-        CHK_STATUS(createRtpRollingBuffer(DEFAULT_ROLLING_BUFFER_DURATION_IN_SECONDS * HIGHEST_EXPECTED_BIT_RATE / 8 / DEFAULT_MTU_SIZE,
-                                          &pKvsRtpTransceiver->sender.packetBuffer));
+        UINT64 size = (UINT64) (pKvsRtpTransceiver->rollingBufferDurationSec * pKvsRtpTransceiver->rollingBufferBitratebps / 8 / DEFAULT_MTU_SIZE);
```

Comment:
(nit) can this variable have a more descriptive name like bufferSize or bufferedPackets - although this is already explained in the log one line below, so not too important.

My justification for still renaming is for when searching for a particular variable in all files when debugging something, can be helpful to have names be more unique to have less generic names like "size" to sift through.